docs/install/_rpi.rst

```diff
@@ -0,0 +1,55 @@
+.. _rpi:
+
+Raspbian
+========
+
+Compatibility
+-------------
+
+Kolibri currently doesn't work out of the box for Raspbian Jessie. We are refining our distribution to work out of the box, but you need to follow the below steps in order to install it (tested on Kolibri 0.9).
+
+Install
+-------
+
+1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+
+      sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
+      sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
+      sudo apt update
+      sudo apt install kolibri
+
+2. Kolibri does not start after installation. This is because ``python3-cffi`` is outdated on Raspbian. Upgrade it like this::
+
+      sudo apt install libffi-dev
```

Comment:
Not sure why we need `libffi-dev`... I tried the installation steps without this step, and kolibri works fine.

Reply:
Sounds good! :+1: 

@indirectlylit Ready for the final review! :wink: